_partials/_migrate_using_postgres_copy.md

```diff
@@ -0,0 +1,26 @@
+<Procedure>
+
+### Restoring data into Timescale with COPY
+
+1.  Connect to your Timescale database:
+
+    ```sql
+    psql "postgres://tsdbadmin:<PASSWORD>@<HOST>:<PORT>/tsdb?sslmode=require"
+    ```
+
+1.  Restore the data to your Timescale database:
+
+    ```sql
+    \COPY <TABLE_NAME> FROM '<TABLE_NAME>.csv' WITH (FORMAT CSV);
```

Comment:
Postgres uses lowercase for the `psql` commands. We should probably do the same.

```suggestion
    \copy <TABLE_NAME> from '<TABLE_NAME>.csv' with (format csv);
```

Reply:
done.